projects/skia/Dockerfile

```diff
@@ -80,38 +80,7 @@ COPY build.sh $SRC/
 COPY skia.diff $SRC/skia/skia.diff
 RUN git apply skia.diff
 
-COPY region_deserialize.options $SRC/skia/region_deserialize.options
```

Comment:
Can you git pull, git rebase and reupload, these changes were in last cl
https://github.com/google/oss-fuzz/blob/master/projects/skia/Dockerfile

Reply:
Sure, I am working on that